corehq/apps/app_manager/templates/app_manager/v2/partials/releases_deploy_modal.html

```diff
@@ -7,14 +7,14 @@
             <button type="button" class="close" data-dismiss="modal">
                 <span aria-hidden="true">&times;</span>
             </button>
-            <h4 class="modal-title">{% trans "Deploying version" %} <span data-bind="text: version"></span></h4>
+            <h4 class="modal-title">{% trans "Making new version" %} <span data-bind="text: version"></span></h4>
```

Comment:
Oops, I confused this with the message that shows while the new version is building. Updated to just say `Version {xyz}`.